packages/commonwealth/client/scripts/views/pages/discussions/DiscussionsPage.tsx

```diff
@@ -193,6 +193,7 @@ const DiscussionsPage = ({ topicName }: DiscussionsPageProps) => {
       topicNameFromURL &&
       topicNameFromURL !== 'archived' &&
       topicNameFromURL !== 'overview' &&
+      topicNameFromURL !== 'undefined' &&
```

Comment:
what if the topic is named undefined 😄

Reply:
you are right, changed it so function returns actually undefined instead of a string